narwhals/_duckdb/utils.py

```diff
@@ -120,6 +121,19 @@ def native_to_narwhals_dtype(duckdb_dtype: str, version: Version) -> DType:
         return dtypes.Boolean()
     if duckdb_dtype == "INTERVAL":
         return dtypes.Duration()
+    if duckdb_dtype.startswith("ENUM"):
+        if version is Version.V1:  # pragma: no cover
+            msg = "Converting to Enum is not supported in narwhals.stable.v1"
+            raise NotImplementedError(msg)
```

Comment:
I mean in principle we have Enum in v1 but we cannot store the categories. Not sure what we should do in this case

Reply:
i think what you'd done is fine